jest.config.js

```diff
@@ -8,7 +10,7 @@ module.exports = {
   transform: {
     '.+\\.(t|j)sx?$': [
       // this matches our SWC options used in https://github.com/vercel/next.js/blob/canary/packages/next/taskfile-swc.js
-      '@swc/jest',
+      path.join(__dirname, './packages/next/jest.js'),
       {
         sourceMaps: 'inline',
```

Comment:
Should we remove the below configuration as it should be handled by `next-swc` now?

Reply:
It will need to be handled by the jest transform to match behavior of the `next-swc-loader` but yeah, good call out 👍